R4ML/R/ml.model.pca.R

```diff
@@ -76,6 +76,7 @@ setClass(
 #'
 #' }
 #'
+#' @import methods
```

Comment:
I think for methods and stats import we can add in the DESCRIPTION file rather than each file